sdk/lib/crt/stdlib/atexit.c

```diff
@@ -69,16 +79,14 @@ _onexit_t CDECL _onexit(_onexit_t func)
   {
     _onexit_t *newtable;
     TRACE("expanding table\n");
-    newtable = calloc(atexit_table_size + 32, sizeof(void *));
+    newtable = realloc(atexit_table, (atexit_table_size + 32) * sizeof(_onexit_t));
```

Comment:
Missing zero-init

Reply:
Again, what is zero-init needed for?

Reply:
My mistake, didn't see there's `atexit_registered` in addition to `atexit_table_size`.